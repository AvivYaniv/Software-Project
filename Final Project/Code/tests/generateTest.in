solve tests/b3_2_empty.txt
generate 4 4
edit tests/b3_3_empty.txt
generate 81 1 #the chances for success are low. this should fail. 
undo
redo
print_board
generate 0 0 adfas
undo
print_board
edit tests/b2_2.txt
generate 17 4
generate 4 17
generate 4 a
generate b 7
generate 1
generate
generate  3 3
generate  17 3
exit