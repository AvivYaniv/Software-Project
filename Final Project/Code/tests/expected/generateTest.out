Sudoku
------
Enter your command:
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
|        |        |        |
|        |        |        |
|        |        |        |
----------------------------
Enter your command:
ERROR: invalid command
Enter your command:
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
Enter your command:
Error: puzzle generator failed
Enter your command:
Error: no moves to undo
Enter your command:
Error: no moves to redo
Enter your command:
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
Enter your command:
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
Enter your command:
Error: no moves to undo
Enter your command:
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
|            |            |            |
|            |            |            |
|            |            |            |
----------------------------------------
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
Error: value not in range 0-11
Enter your command:
Error: value not in range 0-11
Enter your command:
Error: value not in range 0-11
Enter your command:
Error: value not in range 0-11
Enter your command:
ERROR: invalid command
Enter your command:
ERROR: invalid command
Enter your command:
Error: board is not empty
Enter your command:
Error: value not in range 0-11
Enter your command:
Exiting...
